client/web/src/auth/PostSignUpPage.tsx

```diff
@@ -55,14 +64,26 @@ export const PostSignUpPage: FunctionComponent<PostSignUpPage> = ({
 
     const goToSearch = (): void => history.push(getReturnTo(location))
 
+    useEffect(() => {
+        eventLogger.log(getPostSignUpEvent('Page_Viewed'))
+    }, [])
+
     // if the welcome flow was already finished - navigate to search
     if (didUserFinishWelcomeFlow) {
         goToSearch()
     }
 
-    const finishWelcomeFlow = (event: React.MouseEvent<HTMLElement>): void => {
+    const finishWelcomeFlow: FinishWelcomeFlow = (event, { eventName, tabNumber }) => {
         event.currentTarget.blur()
         setUserFinishedWelcomeFlow(true)
+
+        const fullEventName = getPostSignUpEvent(eventName)
+        if (tabNumber) {
+            eventLogger.log(fullEventName, { tabNumber })
```

Comment:
If you want this tabNumber property (or any property) available in our analytics systems, you'll need to pass it to the `publicArgument` argument.
```suggestion
            eventLogger.log(fullEventName, { tabNumber }, { tabNumber })
```